### ISSUE TITLE
Temperature scaling

### ISSUE THREAD

**Issue by XZLeo** [MiraPurkrabek/ProbPose_code#14]
Hi, thank you for your excellent work again! 

I am trying to find the code for temperature scaling. I only find this line:
[Sparsemax](https://github.com/MiraPurkrabek/ProbPose_code/blob/main/mmpose/models/heads/hybrid_heads/probmap_head.py#L642)

`self.temperature = 0.5` is hardcoded. Is this learned on the validation set? 

If I am looking at the wrong place, can you kindly point me to the correct lines? 
Do you find applying Calibration on the probability maps and the presence probability improves the Keypoint localization accuracy?

Thank you very much!

**Comment by MiraPurkrabek** [comment]
Hi @XZLeo ,

Thanks for the question. The code for temperature scaling is not part of the repo. The hard-coded value 0.5 is fine-tuned for CropCOCO dataset but of course, should be updated for other datasets.

To fine-tune temperature for your own dataset, freeze all other parameters and minimize distributions difference.

**Comment by XZLeo** [comment]
Thanks for the answer. Can you also share the configuration file for testing on OCHuman? 

Both ProbPose-DP and ProbPose are trained only on COCO and tested on OCHuman. Do I understand correctly?

**Comment by MiraPurkrabek** [comment]
Hi, 

Yes, trained on COCO with crop augmentation, tested on COCO val, CropCOCO and OCHuman.

To test on OCHuman, simply replace (or expand) the dataset definition in config. Example is [here](https://github.com/MiraPurkrabek/BBoxMaskPose/blob/BMP-v2/mmpose%2Fconfigs%2FProbMaskPose%2FPMPose-b-1.0.0.py#L208).
